test/acceptance/test/add_tests.go

```diff
@@ -275,122 +226,22 @@ var _ = Describe("Weave GitOps Add App Tests", func() {
-			}
-		})
-
-	})
-
-	It("Test1 - Verify that gitops can deploy an app with different config-repo and add repo", func() {
-		var repoAbsolutePath string
-		var configRepoRemoteURL string
-		private := true
-		tip := generateTestInputs()
-		appName := tip.appRepoName
-		appConfigRepoName := "config-repo-" + RandString(8)
-		appRepoRemoteURL := "https://github.com/" + githubOrg + "/" + tip.appRepoName + ".git"
-		configRepoRemoteURL = "https://github.com/" + githubOrg + "/" + appConfigRepoName + ".git"
-
-		addCommand := "add app --url=" + appRepoRemoteURL + " --auto-merge=true"
-
-		defer deleteRepo(tip.appRepoName, gitproviders.GitProviderGitHub, githubOrg)
-		defer deleteRepo(appConfigRepoName, gitproviders.GitProviderGitHub, githubOrg)
-		defer deleteWorkload(tip.workloadName, tip.workloadNamespace)
-
-		By("And application repo does not already exist", func() {
-			deleteRepo(tip.appRepoName, gitproviders.GitProviderGitHub, githubOrg)
-			deleteRepo(appConfigRepoName, gitproviders.GitProviderGitHub, githubOrg)
-		})
-
-		By("And application workload is not already deployed to cluster", func() {
-			deleteWorkload(tip.workloadName, tip.workloadNamespace)
-		})
-
-		By("When I create a private repo for gitops app config", func() {
-			appConfigRepoAbsPath := initAndCreateEmptyRepo(appConfigRepoName, gitproviders.GitProviderGitHub, private, githubOrg)
-			gitAddCommitPush(appConfigRepoAbsPath, tip.appManifestFilePath)
-		})
-
-		By("When I create a private repo with my app workload", func() {
-			repoAbsolutePath = initAndCreateEmptyRepo(tip.appRepoName, gitproviders.GitProviderGitHub, private, githubOrg)
-			gitAddCommitPush(repoAbsolutePath, tip.appManifestFilePath)
-		})
-
-		By("And I install gitops to my active cluster", func() {
-			installAndVerifyWego(WEGO_DEFAULT_NAMESPACE, configRepoRemoteURL)
+		By("And app should not get deleted until the PR is merged", func() {
+			err = waitForAppRemoval(appName, EVENTUALLY_DEFAULT_TIMEOUT)
+			Expect(err).To(HaveOccurred())
 		})
 
-		By("And I run gitops add command with --url", func() {
-			runWegoAddCommand(repoAbsolutePath+"/../", addCommand, WEGO_DEFAULT_NAMESPACE)
+		By("When I merge the PR for app deletion", func() {
+			mergePR(repoAbsolutePath, prLink, gitProvider)
 		})
 
-		By("Then I should see my workload deployed to the cluster", func() {
-			verifyWegoAddCommand(appName, WEGO_DEFAULT_NAMESPACE)
-			verifyWorkloadIsDeployed(tip.workloadName, tip.workloadNamespace)
+		By("Then app should get deleted from the cluster", func() {
+			err = waitForAppRemoval(appName, THIRTY_SECOND_TIMEOUT)
+			Expect(err).NotTo(HaveOccurred())
 		})
 	})
 
-	It("Test2 - Verify that gitops can deploy and remove a gitlab app with different config-repo and add repo", func() {
+	It("Test1 - Verify that gitops can deploy and delete an app with separate config-repo over https protocol", func() {
```

Comment:
I see a `Test1` above, is this a typo?

Reply:
No, the Test1, Test2 tags are for the different acceptance test jobs:

<img width="967" alt="image" src="https://user-images.githubusercontent.com/76788247/152812102-87fe078c-f687-4ba5-b098-8084c39407ba.png">

Reply:
ahh gotcha! Thanks